octobot_trading/modes/channel/abstract_mode_producer.py

```diff
@@ -83,46 +107,109 @@ async def start(self) -> None:
+           len(self.time_frame_filter) < len(trigger_time_frames):
+            missing_time_frames = [tf for tf in trigger_time_frames if tf not in self.time_frame_filter]
+            self.logger.error(f"Missing timeframe to satisfy {trigger_time_frames} required time frames. "
+                              f"Please activate those timeframes {missing_time_frames}")
+        self.matrix_id = exchanges.Exchanges.instance().get_exchange(self.exchange_manager.exchange_name,
+                                                                     self.exchange_manager.id).matrix_id
+        for registration_topic in registration_topics:
+            if registration_topic == channels_name.OctoBotEvaluatorsChannelsName.MATRIX_CHANNEL.value:
+                try:
+                    import octobot_evaluators.evaluators.channel as evaluators_channel
+                    import octobot_evaluators.enums as evaluators_enums
+                    consumer = await evaluators_channel.get_chan(registration_topic, self.matrix_id).new_consumer(
+                        callback=self.get_callback(registration_topic),
+                        priority_level=self.priority_level,
+                        matrix_id=self.matrix_id,
+                        cryptocurrency=currency_filter,
+                        symbol=symbol_filter,
+                        evaluator_type=evaluators_enums.EvaluatorMatrixTypes.STRATEGIES.value,
+                        exchange_name=self.exchange_name,
+                        # no time_frame filter to allow receiving updates from strategies without timeframes in wildcard
+                        time_frame=common_constants.CONFIG_WILDCARD if self.is_time_frame_wildcard()
+                        else self.time_frame_filter,
+                        supervised=self.exchange_manager.is_backtesting
+                    )
+                    self.evaluator_consumers.append(
+                        (consumer, registration_topic)
+                    )
+                    self.trading_mode.is_triggered_after_candle_close = False
+                except (KeyError, ImportError):
+                    self.logger.error(f"Can't connect matrix channel on {self.exchange_name}")
+            else:
+                # trading channels
+                consumer = await exchanges_channel.get_chan(registration_topic, self.exchange_manager.id).new_consumer(
+                    callback=self.get_callback(registration_topic),
+                    priority_level=self.priority_level,
+                    cryptocurrency=currency_filter,
+                    symbol=symbol_filter,
+                    time_frame=self.time_frame_filter
+                )
+                self.trading_consumers.append(
+                    (consumer, registration_topic)
+                )
+                if registration_topic == channels_name.OctoBotTradingChannelsName.OHLCV_CHANNEL.value:
+                    self.trading_mode.is_triggered_after_candle_close = True
```

Comment:
Could we create a method to separate this code from filter definition (to simplify this method), like `_subscribe_to_registration_topic`?